capstone-app/src/app/modules/history/history.module.ts

```diff
@@ -3,10 +3,12 @@ import { CommonModule } from '@angular/common';
 import { WikiMediaComponent } from './wiki-media/wiki-media.component';
 import { FormsModule } from '@angular/forms';
 import { HttpClientModule } from '@angular/common/http';
+import { MatCardModule } from '@angular/material/card';
+import { MaterialModule } from './../material/material.module';
```

Comment:
So if you import the material module you don't need to import MatCard. Can you import MatCard there and then add Material module to the list of imports here?